src/js/common/components/Widgets/ReadMore.jsx

```diff
@@ -27,6 +30,43 @@ export default class ReadMore extends Component {
   toggleLines (event) {
     event.preventDefault();
     const { readMore } = this.state;
+    const { location: { pathname: currentPathname } } = window;
+    const currentPage = lookupPageNameAndPageTypeDict(currentPathname);
+
+    const {
+      eventName = 'clickShowMoreAboutPolitician',
+      politicianIdShowMore,
+      politicianNameShowMore,
+    } = this.props;
+
+    console.log("Right here!",readMore, politicianNameShowMore, politicianIdShowMore);
+
+    TagManager.dataLayer({
+    dataLayer: {
+      event: 'click',
+      actionDetails: {
+        actionType: 'showMore',
```

Comment:
For actionType, let's dynamically set 'showMore' when showing more text and 'showLess' when showing less text.